lib/components/Atom/Calendar/index.tsx

```diff
@@ -0,0 +1,126 @@
+import { useEffect } from "react";
+import clsx from "clsx";
+import Calendar from "react-calendar";
+import "react-calendar/dist/Calendar.css";
+import "./date-picker.scss";
+import {
+    LabelPairedChevronRightSmFillIcon,
+    LabelPairedChevronLeftSmFillIcon,
+} from "@deriv/quill-icons";
+import { Text } from "@components/Typography";
+
+interface DatePicker
+    extends Omit<React.ComponentProps<typeof Calendar>, "onChange"> {
+    optionsConfig?: Record<string, string>;
+    onChange?: (
+        value: Value,
+        event?: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+    ) => void;
+    onFormattedDate?: (value: string) => void;
+    isWidthFixed?: boolean;
+    wrapperClassName?: string;
+}
+
+type ValuePiece = Date | null;
+
+type Value = ValuePiece | [ValuePiece, ValuePiece];
+
+export const DatePicker = ({
+    activeStartDate,
```

Comment:
Any difference if we keep this `startDate` ?

Reply:
No, it's okay to rename, I'll push the commit